core/platform/taskqueue/dev_mode_taskqueue_services_test.py

```diff
@@ -43,7 +49,12 @@ def test_creating_dev_mode_task_will_create_the_correct_post_request(self):
         correct_task_name = 'task1'
 
         def mock_create_task(
-                queue_name, url, payload, scheduled_for=None, task_name=None): # pylint: disable=unused-argument
+                queue_name: str,
+                url: str,
+                payload: Optional[Dict[str, Any]] = None,
```

Comment:
Just for this you can add a comment above that `Dict[str, Any]` is needed because it emulates the behaviour of `<module>.<class>.<method>`.

Reply:
Done

Reply:
Wait, why is it optional here?

Reply:
The default value of payload here is None. So it should definitely be Optional.

Reply:
Yeah, but what about the usages of this function. I think we should maybe prefer `{}` and pass it explicitly everywhere.

Reply:
Yes this is only used in test files. Done.